### PR TITLE
added get_events, update_events, updated get_v2_events

### PR DESCRIPTION
**Pull request by nfalke-**

### CODE REVIEW

File: riskiq/api.py [hunk 1]
```diff
@@ -1002,41 +1004,42 @@ def post_inventory_search(self, query=None, filter=None, filters=None):
 
     def get_v2_events(self, days=1, start=None, end=None, count=50, offset=0):
         """
-        Get inventory items for workspace.  Can behave in one of 3 ways:
```
Comment (jonasedgeworth) [inline riskiq/api.py:1005]: Appears to be a c&p issue. Please update the comment appropriately.

File: riskiq/api.py [hunk 2]
```diff
@@ -1002,41 +1004,42 @@ def post_inventory_search(self, query=None, filter=None, filters=None):
 
     def get_v2_events(self, days=1, start=None, end=None, count=50, offset=0):
```
Comment (jonasedgeworth) [inline riskiq/api.py:1005]: What is the difference between a V1 event and a V2 event? There should only be Events (no versions), and hence there should be no _v2_ in the method name.

File: riskiq/api.py [hunk 3]
```diff
@@ -1002,41 +1004,42 @@ def post_inventory_search(self, query=None, filter=None, filters=None):
...
-            ]
-        }
-        return self._post('event', 'search', event_filter, count=count,
-                          offset=offset)
+        event_filter = SearchFilter(field="createdAt", value=start, op="GTE")
+        return self.get_events(event_filter.asdict(), count=count, offset=offset)
+
+    def get_events(self, event_filter, count=50, offset=0):
+        '''
+        Get inventory items for workspace
```
Comment (jonasedgeworth) [inline riskiq/api.py:1028]: Again, inventory comment referenced in a event method.

**Comment by johannestaas** [comment]
Should be fixed in latest commits. This was from an old merge. I'm trying to stick to get_ and post_ for the calls to the API that don't do more than make a simple call to the endpoint with the data provided, so   I renamed to search_events. Definitely agree, _v2_ should've been dropped.

**Comment by johannestaas** [comment]
honestly I want to completely drop search_inventory and search_events since they perform extra logic that can be easily implemented by the user calling the post_inventory_search and post_event_search.

These two functions don't behave like the rest of the client.

**Comment by johannestaas** [comment]
Seems like there are multiple issues. One docstring for the end keyword argument even specified "does nothing, left in for legacy code" and it ends up possibly affecting the results. Deleted both search_ functions.

I think this extra logic should be left out of the client for now while the rest are get_ and post_ methods directly hitting the API. The SearchFilter class is enough to make it more convenient to build filters, and that should be left to the user.

**Pull request merged**